packages/contracts/script/deploy/optimism/L1Resolver.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {IGatewayVerifier} from "@unruggable/gateways/IGatewayVerifier.sol";
+import {L1Resolver} from "../../../src/optimism/L1Resolver.sol";
+import {DeployHelper} from "../../DeployHelper.sol";
+
+contract OPResolverScript is DeployHelper {
+
+    function run() external {
+        IGatewayVerifier verifier =
+            IGatewayVerifier(0x5F1681D608e50458D96F43EbAb1137bA1d2A2E4D);
```

Comment:
Not a big fan of this hardcoded address but i see that it will probably never be changed. So i understand why to do this way also heheh.